<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="PRASHANT" failures="3" tests="5" name="Test" time="38.262" errors="0" timestamp="2024-06-15T08:15:28 IST">
  <testcase classname="com.Scenario1.OHRM_Scenario1" name="loginTest" time="8.594"/>
  <testcase classname="com.Scenario1.OHRM_Scenario1" name="loginTest" time="5.673">
    <failure type="java.lang.AssertionError" message="Login Fail expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: Login Fail expected [true] but found [false]
at com.Scenario1.OHRM_Scenario1.loginTest(OHRM_Scenario1.java:29)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 30 stack frames]]>
    </failure>
  </testcase> <!-- loginTest -->
  <testcase classname="com.Scenario1.OHRM_Scenario1" name="loginTest" time="7.613">
    <failure type="java.lang.AssertionError" message="Login Fail expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: Login Fail expected [true] but found [false]
at com.Scenario1.OHRM_Scenario1.loginTest(OHRM_Scenario1.java:29)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 30 stack frames]]>
    </failure>
  </testcase> <!-- loginTest -->
  <testcase classname="com.Scenario1.OHRM_Scenario1" name="loginTest" time="9.137"/>
  <testcase classname="com.Scenario1.OHRM_Scenario1" name="loginTest" time="6.351">
    <failure type="java.lang.AssertionError" message="Login Fail expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: Login Fail expected [true] but found [false]
at com.Scenario1.OHRM_Scenario1.loginTest(OHRM_Scenario1.java:29)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 30 stack frames]]>
    </failure>
  </testcase> <!-- loginTest -->
</testsuite> <!-- Test -->
